lib/src/util/io.dart

```diff
@@ -130,8 +130,12 @@ void warn(String message, {bool color}) {
 /// applications that don't print out the bound port.
 Future<T> getUnusedPort<T>(FutureOr<T> tryPort(int port)) async {
   T value;
+  var supportsIPv6 = false;
+  if (await _supportsIPv6()) {
+    supportsIPv6 = true;
+  }
```

Comment:
Removed the unnecessary conditional. I think it makes sense to be in here as I don't want getUnsafedUnusedPort to make multiple binds since it is called in a while.